docs/Version-Comparison.md

```diff
@@ -92,13 +66,12 @@ maintain them.
 | Plugin Sandbox Extensions                         | no                                                                                                          | yes ([See extensions](SandboxExtension.md))                                                     |
 | Data storage                                      | local, S3 (elevation only)                                                                                  | extensible with local, ZIP,<br>and Google Cloud plugins, S3 available                           |
 | Transfer Priority                                 | yes                                                                                                         | yes                                                                                             |
-| REST API format                                   | XML, JSON                                                                                                   | JSON only                                                                                       |
 
 ## Commentary on OTP1 features removed from OTP2
 
 OTP2 brings significant improvements in speed and scalability, but does not retain all features of
-OTP1. We have chosen to prioritize long-term maintainability, so only those features that are "
-owned" by a team of professional developers will be carried over to OTP2.
+OTP1. We have chosen to prioritize long-term maintainability, so only those features that are 
+"owned" by a team of professional developers will be carried over to OTP2.
 
 Features that have been removed to simplify the code base and improve maintainability may be removed
```

Comment:
This sentence is a little weird: is the feature removed or not?

Reply:
I've rewritten this entire paragraph which was indeed strange: 9d6adbfa2f8d1e6345aded14132c8d6dfb79ee61